Description: Disable update check as it's useless for a packaged version
Author: Jakob Haufe <sur5r@sur5r.net>

diff --git a/src/MainWindow.cpp b/src/MainWindow.cpp
index e5d78d2..581a168 100644
--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -130,7 +130,7 @@ MainWindow::MainWindow() :
     mouseTimer->setSingleShot(true);
     connect(mouseTimer, SIGNAL(timeout()), SLOT(hideMouse()));
 
-    QTimer::singleShot(0, this, SLOT(checkForUpdate()));
+    //QTimer::singleShot(0, this, SLOT(checkForUpdate()));
 
 }
 
